flowbased-computation/flowbased-computation-impl/pom.xml

```diff
@@ -37,6 +37,11 @@
         </dependency>
 
         <!-- Test dependencies -->
+        <dependency>
+            <groupId>com.powsybl</groupId>
+            <artifactId>powsybl-config-test</artifactId>
+            <scope>test</scope>
+        </dependency>
```

Comment:
Please respect the alphabet...

Reply:
I will respect the alphabet when the alphabet respects me !